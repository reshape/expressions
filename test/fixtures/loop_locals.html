<p>x</p>
<each loop='item, index of items'>
  <p>{{index}}, {{item}}, {{ foo }}</p>
</each>
<p>x</p>
