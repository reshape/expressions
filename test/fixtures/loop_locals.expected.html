<p>x</p>

  <p>0, 1, bar</p>

  <p>1, 2, bar</p>

  <p>2, 3, bar</p>

<p>x</p>
